modules/iam-assumable-role/variables.tf

```diff
@@ -131,9 +131,8 @@ variable "role_description" {
   default     = ""
 }
 
-variable "role_sts_externalid" {
-  description = "STS ExternalId condition value to use with a role (when MFA is not required)"
-  type        = string
-  default     = null
+variable "role_sts_external_ids" {
+  description = "STS External ID condition values to use with a role (when MFA is not required)"
+  type        = list(string)
```

Comment:
Please update the code to have the same variable name as before but with `type = any` to be able to support (string and list(string) values simultaneously).

Reply:
Sure thing @antonbabenko.  I have reverted back to the original variable name.